linchpin/api/__init__.py

```diff
@@ -145,6 +142,7 @@ def run_playbook(self, pinfile, targets='all', playbook='up'):
                                                 console=self.console)
```

Comment:
@smatula I've updated the PR as you mentioned. I agree, this is better. LMK what you think.